sleap/gui/widgets/video.py

```diff
@@ -1732,6 +1733,211 @@ def updateEdge(self, node: QtNode):
+        self.top_right_box.setPen(box_pen)
+        self.top_right_box.setBrush(box_brush)
+        self.top_right_box.setOpacity(0.8)
+        self.top_right_box.setCursor(QCursor(Qt.DragMoveCursor))
+
+        self.bottom_left_box = QtWidgets.QGraphicsRectItem(parent=self)
+        self.bottom_left_box.setPen(box_pen)
+        self.bottom_left_box.setBrush(box_brush)
+        self.bottom_left_box.setOpacity(0.8)
+        self.bottom_left_box.setCursor(QCursor(Qt.DragMoveCursor))
+
+        self.bottom_right_box = QtWidgets.QGraphicsRectItem(parent=self)
+        self.bottom_right_box.setPen(box_pen)
+        self.bottom_right_box.setBrush(box_brush)
+        self.bottom_right_box.setOpacity(0.8)
+        self.bottom_right_box.setCursor(QCursor(Qt.DragMoveCursor))
+
+    def setRect(self, rect: QRectF):
+        # Update edge boxes along with instance box
+        super().setRect(rect)
+        x1, y1, x2, y2 = rect.getCoords()
+        w = self.box_width
+        self.top_left_box.setRect(QRectF(QPointF(x1, y1), QPointF(x1 + w, y1 + w)))
+        self.top_right_box.setRect(QRectF(QPointF(x2 - w, y1), QPointF(x2, y1 + w)))
+        self.bottom_left_box.setRect(QRectF(QPointF(x1, y2 - w), QPointF(x1 + w, y2)))
+        self.bottom_right_box.setRect(QRectF(QPointF(x2 - w, y2 - w), QPointF(x2, y2)))
+
+    def mousePressEvent(self, event):
+        """Custom event handler for pressing on an adjustable corner box.
+
+        This function recognizes that the user has begun resizing the instance and
+        stores relevant information about the bounding box before the transformation.
+        """
+        if event.button() == Qt.RightButton:
+            self.resizing = None
+        else:
+            # Check if the user clicked on one of the edge boxes
+            if self.top_left_box.contains(event.pos()):
+                self.resizing = "top_left"
+                self.origin = self.rect().bottomRight()
+            elif self.top_right_box.contains(event.pos()):
+                self.resizing = "top_right"
+                self.origin = self.rect().bottomLeft()
+            elif self.bottom_left_box.contains(event.pos()):
+                self.resizing = "bottom_left"
+                self.origin = self.rect().topRight()
+            elif self.bottom_right_box.contains(event.pos()):
+                self.resizing = "bottom_right"
+                self.origin = self.rect().topLeft()
+            else:
+                self.resizing = None
+
+        self.ref_width = self.rect().width()
+        self.ref_height = self.rect().height()
+
+    def mouseMoveEvent(self, event):
+        """Custom event handler for moving an adjustable corner box.
+
+        This function resizes the bounding box as the user drags one of its corners.
+        """
+        # Scale the bounding box and QtInstance if an edge box is selected
+        if event.button() == Qt.RightButton:
+            pass
+        else:
```

Comment:
This is a weird way of selecting for the left button -- It also will fire when you use the middle or side buttons.

How about:
```suggestion
        if event.button() == Qt.LeftButton:
```